### PR TITLE
feat: updated about.md.

### PR DESCRIPTION
**Pull request by C-nit**

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -13,5 +13,5 @@ Follow our [organization](https://github.com/PaccMann) on GitHub for the latest
 If you use PaccMann in your research, please consider citing these papers:
 
 - PaccMann predictor [paper](https://pubs.acs.org/doi/full/10.1021/acs.molpharmaceut.9b00520)
```
Comment (jannisborn) [inline about.md:15]:
```suggestion
- PaccMann [paper](https://pubs.acs.org/doi/full/10.1021/acs.molpharmaceut.9b00520) (predicting anticancer drug sensitivity)
```

File: about.md [hunk 2]
```diff
@@ -13,5 +13,5 @@ Follow our [organization](https://github.com/PaccMann) on GitHub for the latest
...
-- PaccMann<sup>RL</sup> [paper](https://arxiv.org/abs/1909.05114)
-
+- PaccMann<sup>RL</sup> [paper](https://doi.org/10.1016/j.isci.2021.102269) (generating compounds given a transcriptomic profile, case study on lung cancer)
```
Comment (jannisborn) [inline about.md:16]:
```suggestion
- PaccMann webservice [paper](https://doi.org/10.1093/nar/gkaa327)
- PaccMann<sup>RL</sup> [paper](https://doi.org/10.1016/j.isci.2021.102269) (generating novel molecules given a transcriptomic profile)
```

File: about.md [hunk 3]
```diff
@@ -13,5 +13,5 @@ Follow our [organization](https://github.com/PaccMann) on GitHub for the latest
...
+- PaccMann<sup>RL</sup> [paper](https://doi.org/10.1088/2632-2153/abe808) (generating ligands given a protein sequence, case study on SARS-CoV-2)
```
Comment (jannisborn) [inline about.md:17]:
```suggestion
- PaccMann<sup>RL</sup> for proteins [paper](https://doi.org/10.1088/2632-2153/abe808) (generating novel ligands given a protein sequence, case study on SARS-CoV-2)
```

**Review by jannisborn — APPROVED** [review]
Thanks a lot, I also added the NAR paper and made some minor suggestions

**Comment by C-nit** [comment]
thanks for the additions.

**Pull request merged**